### ISSUE TITLE
Heading semantics askew due to site title (h1) placement

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/foundation#79]
The site title, which is correctly formatted as `h1`, is currently nestled under the `nav` element in the `header`:
<img width="384" alt="Screenshot of page source which identifies <h1 class='site-title'>'s location" src="https://github.com/user-attachments/assets/71212d73-e9c7-491d-9df8-6295b23731e4" />
This is skewing overall semantics and may impact screen readers and other tools that rely on proper semantic ordering as the site title is not currently being recognized as a top-level heading.

I would suggest relocating the site title similar to the following model (while maintaining the `h1` rank), which positions the title element as a child to the `div` container in the `header` rather than the `nav`:
<img width="1265" alt="Screenshot of page with h1 site title identified by browser dev tools, which are open to portion of page source with <h1 class='site-title'>'s location reordered" src="https://github.com/user-attachments/assets/121752a1-dd8c-4366-87f7-47cbd7d52db5" />
A change like this allows parsing to properly identify this element as top-level in respect to other elements.

**Comment by kimisgold** [comment]
While doing some reorganizing in the header, I also moved the site title out of the nav element. It should be ready for testing on the `master` branch.

**Comment by aviannamiller** [comment]
Wonderful! That is now picking up nicely in the hierarchy.
